components/arm/fake/fake.go

```diff
@@ -256,8 +271,8 @@ func (a *Arm) Geometries(ctx context.Context, extra map[string]interface{}) ([]s
 
 func modelFromName(model, name string) (referenceframe.Model, error) {
 	switch model {
-	case xarm.ModelName6DOF, xarm.ModelName7DOF, xarm.ModelNameLite:
-		return xarm.MakeModelFrame(name, model)
+	case xArmModelName6DOF, xArmModelName7DOF, xArmModelNameLite:
```

Comment:
I'm not sure what the point of fake arm is anymore and why we need to continue support for xarms here

Reply:
What if a user does not own an xarm in real life but would like to emulate its behavior?

Suppose a user is building an arm application which requires them to navigate to specific points on a table, e.g. pick and place, and that it is crucial that the arm is able to reach all points on the table. 

A user may then use the fake arm which emulates other arm drivers to see if all points are then reachable through the use of either `MoveToEndPositions` or `motionservice.Move`. In this example given that the table is "large" an xarm lite6 would likely not suffice but an xarm6 might.

Overall, I think the original intention of having a fake arm still stands despite the team's effort to modularize arm drivers.

Please lmk your thoughts, perhaps there is an angle I am not considering.

Reply:
You can still provide it as a file the same as you would do for any other model

Reply:
hmmm, I just realized that I will need to think about how removing "arm model" would impact how the wrapper driver currently works too.

Reply:
Update, I think removing arm models from RDK breaks the wrapper arm since we do this https://github.com/viamrobotics/rdk/blob/main/components/arm/wrapper/wrapper.go#L85

Reply:
I said this before in person, but I don't think this needs to be addressed in this PR as it is not part of its scope